packages/rancher-istio/charts/1.6.400/configs/istio-base.yaml

```diff
@@ -0,0 +1,251 @@
+apiVersion: install.istio.io/v1alpha1
+kind: IstioOperator
+spec:
+  addonComponents:
+    grafana:
+      enabled: {{ .Values.grafana.enabled }}
+    istiocoredns:
+      enabled: {{ .Values.istiocoredns.enabled }}
+    kiali:
+      enabled: {{ .Values.kiali.enabled }}
+    prometheus:
+      enabled: {{ .Values.prometheus.enabled }}
+    tracing:
+      enabled: {{ .Values.tracing.enabled }}
+  components:
+    base:
+      enabled: {{ .Values.base.enabled }}
+    citadel:
+      enabled: {{ .Values.citadel.enabled }}
+    cni:
+      enabled: {{ .Values.cni.enabled }}
+    egressGateways:
+    - enabled: {{ .Values.egressGateways.enabled }}
+      name: istio-egressgateway
+    ingressGateways:
+    - enabled: {{ .Values.ingressGateways.enabled }}
+      name: istio-ingressgateway
+    istiodRemote:
+      enabled: {{ .Values.istiodRemote.enabled }}
+    pilot:
+      enabled: {{ .Values.pilot.enabled }}
+    policy:
+      enabled: {{ .Values.policy.enabled }}
+    telemetry:
+      enabled: {{ .Values.telemetry.enabled }}
+  hub: docker.io/istio
+  meshConfig:
+    {{- if .Values.global.accessLogFile }}
+    accessLogFile: "{{ .Values.global.accessLogFile }}"
+    {{- end }}
+    {{- if .Values.global.disablePolicyChecks }}
+    disablePolicyChecks: {{ .Values.global.disablePolicyChecks }}
+    {{- end }}
+    defaultConfig:
+      proxyMetadata: {}
+    {{- if .Values.global.enableAutoMtls }}
+    enableAutoMtls: {{ .Values.global.enableAutoMtls }}
+    {{- end }}
+  profile: default
+  tag: 1.6.4
+  values:
+    global:
+      arch:
+        {{- if .Values.global.arch.amd64 }}
+        amd64: {{ .Values.global.arch.amd64 }}
+        {{- end }}
+        {{- if .Values.global.arch.ppc64le }}
+        ppc64le: {{ .Values.global.arch.ppc64le }}
+        {{- end }}
+        {{- if .Values.global.arch.s390x }}
+        s390x: {{ .Values.global.arch.s390x }}
+        {{- end }}
+      {{- if .Values.global.configValidation }}
+      configValidation: {{ .Values.global.configValidation }}
+      {{- end }}
+      {{- if .Values.global.controlPlaneSecurityEnabled }}
+      controlPlaneSecurityEnabled: {{ .Values.global.controlPlaneSecurityEnabled }}
+      {{- end }}
+      {{- if .Values.global.defaultNodeSelector }}
+      defaultNodeSelector: {{ .Values.global.defaultNodeSelector | toJson | indent 4 }}
+      {{- end }}
+      {{- if .Values.global.defaultPodDisruptionBudget.enabled }}
+      defaultPodDisruptionBudget:
+        enabled: {{ .Values.global.defaultPodDisruptionBudget.enabled }}
+      {{- end }}
+      defaultResources:
+        requests:
+          {{- if .Values.global.defaultResources.requests.cpu }}
```

Comment:
Should probably put this if 2 lines above with just ` .Values.global.defaultResources`